Docker/JobDefinitionParser.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Keboola\DockerBundle\Docker;
+
+use Keboola\Syrup\Exception\UserException;
+
+class JobDefinitionParser
+{
+    /**
+     * @var JobDefinition[]
+     */
+    private $jobDefinitions = [];
+
+    /**
+     * @param Component $component
+     * @param array $configData
+     */
+    public function parseConfigData(Component $component, array $configData)
+    {
+        $jobDefinition = new JobDefinition($configData);
+        $jobDefinition->setComponent($component);
+        $this->jobDefinitions = [$jobDefinition];
+    }
+
+    public function parseConfig(Component $component, $config)
+    {
+        $this->jobDefinitions = [];
+        if (count($config['rows']) == 0) {
+            $jobDefinition = new JobDefinition($config['configuration']);
```

Comment:
nemohlo by jobDefinition byt immutable?

Reply:
takže `new JobDefinition($configData, $component)` a žádný settery? potřebuju tomu pak naloadovat další properties (`configId`, `rowId`, ...), to bys tomu taky předal v ctoru jako pole `$options` třeba?

Reply:
dal bych  JobDefintions($configData, $component, $configId, $configVersion, $rowId, $rowVersion, $state, $isDisabled)

Reply:
:+1: